packages/components/src/theme/theme-dark/theme-dark-semantic.css

```diff
@@ -6,6 +6,21 @@
   --dh-color-foreground-hsl: var(--dh-color-white-hsl);
 
   --dh-color-accent-hsl: var(--dh-color-blue-600-hsl);
+  --dh-color-accent-100-hsl: var(--dh-color-blue-100-hsl);
```

Comment:
If we are going to keep the non-hsl `--dh-color-accent-xxx` versions, we should probably update them to derive from these new variables

Reply:
sure